Chart.yaml

```
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "1.0.1"
+appVersion: "1.1.2"
 description: Helm chart to deploy the Elastic operator (ECK / Elastic Cloud on Kubernetes)
 name: elastic-operator
-version: 0.2.0
+version: 0.2.1
```

```suggestion
version: 0.3.0
```

This warrants at least a minor bump :)